pkg/server/api/svid/v1/service.go

```diff
@@ -324,6 +328,96 @@ func (s *Service) NewJWTSVID(ctx context.Context, req *svid.NewJWTSVIDRequest) (
 	}, nil
 }
 
-func (s *Service) NewDownstreamX509CA(context.Context, *svid.NewDownstreamX509CARequest) (*svid.NewDownstreamX509CAResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+func (s *Service) newX509CASVID(ctx context.Context, entry *types.Entry, csrBytes []byte) (resp *svid.NewDownstreamX509CAResponse, err error) {
```

Comment:
nit: this method should be grouped below the exported methods with the other unexported methods.